vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/system.go

```diff
@@ -76,14 +77,23 @@ func (device *Device) DeviceInfo() (*DeviceInfo, error) {
 
 // SetPassword invokes the set password workflow on the device. Should be called only if
 // deviceInfo.Initialized is false.
-func (device *Device) SetPassword() error {
+//
+// `seed_len` must be exactly 16 or 32, creating a 16-byte or a 32-byte seed, corresponding to 12
+// resp. 24 BIP39 recovery words.
+func (device *Device) SetPassword(seedLen int) error {
```

Comment:
@Beerosagos I merged https://github.com/digitalbitbox/bitbox02-api-go/pull/81 to speed this up a little, since you can see the change here to anyway. If something can be improved here I'll open a new PR in bitbox02-api-go to fix it.